packages/venia-ui/lib/components/LegacyMiniCart/productOptions.css

```diff
@@ -12,3 +12,10 @@
     grid-auto-flow: column;
     grid-auto-columns: max-content;
 }
+
+.optionName {
+}
```

Comment:
You can just leave these out of the CSS file if they're empty.

Reply:
I have seen empty CSS previously, so I thought we allow them. Cool will remove it.

Reply:
In the past we have used certain empty rules to allow for the possibility of customization but I don't think they need to be defined in the CSS file. I think @jimbo was just suggesting you remove these from CSS but keep the `className` prop on the components.